dep-graph/dep-graph/src/app/styles-graph/edges.ts

```diff
@@ -41,9 +41,22 @@ const dynamicEdges: Stylesheet = {
   },
 };
 
+const typeOnlyEdges: Stylesheet = {
+  selector: 'edge.typeOnly',
+  style: {
+    label: 'type only',
+    'font-size': '16px',
+    'edge-text-rotation': 'autorotate',
+    'curve-style': 'unbundled-bezier',
+    'line-dash-pattern': [5, 5],
+    'line-style': 'dashed',
+  },
+};
```

Comment:
@philipjfulcher
I noticed you copied the contents of this file over to the React app in #8152. Note that if by some chance my PR is accepted and merged before yours, this edge definition should be copied too.
If yours PR is merged first, I'll add this to the React app.

Reply:
My PR was merged. :) Let me know if you have trouble rebasing this, I can help with conflicts.

Reply:
> My PR was merged. :) Let me know if you have trouble rebasing this, I can help with conflicts.

Seems I was able to rebase it safely. Would you mind reviewing?